Resources/views/Collector/swiftmailer.html.twig

```diff
@@ -1,77 +1,175 @@
+                {% for name in collector.mailers %}
+                    <tr>
+                        <th>{{ name }}{{ collector.isDefaultMailer(name) ? ' (default mailer)' : '' }}</th>
+                        <td>{{ collector.messageCount(name) }} {{ collector.isSpool(name) ? 'spooled' : 'sent' }}</td>
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    {% else %}
+        {% if collector.mailers|length > 1 %}
+            <h3>Messages sent by each mailer</h3>
+
+            <div class="metrics">
+                {% for name in collector.mailers %}
+                    <div class="metric">
+                        <span class="value">{{ collector.messageCount(name) }}
+                        {% if collector.messageCount(name) > 0 %}
+                            <span class="unit">{{ collector.isSpool(name) ? 'spooled' : 'sent' }}</span>
+                        {% endif %}
+                        </span>
+                        <span class="label">
+                            {{ name }} {{ collector.isDefaultMailer(name) ? ' (default app mailer)' : '' }}
+                        </span>
+                    </div>
+                {% endfor %}
+            </div>
+        {% else %}
+            <div class="metrics">
+                {% for name in collector.mailers %}
+                    <div class="metric">
+                        <span class="value">{{ collector.messageCount(name) }}
+                        {% if collector.messageCount(name) > 0 %}
+                            <span class="unit">{{ collector.isSpool(name) ? 'spooled' : 'sent' }}</span>
+                        {% endif %}
+                        </span>
+                        <span class="label">sent messages</span>
+                    </div>
+                {% endfor %}
+            </div>
+        {% endif %}
+    {% endif %}
 
     {% for name in collector.mailers %}
-        <h3>Mailer <em>{{ name }}</em>{{ collector.isDefaultMailer(name) ? ' (default mailer)' : '' }}</h3>
+        {% if collector.mailers|length > 1 %}
+            <h3>
+                {{ name }} <span>mailer</span>
+                <small>{{ collector.isDefaultMailer(name) ? '(default app mailer)' }}</small>
+            </h3>
+        {% endif %}
 
         {% if not collector.messages(name) %}
-            <p>
-                <em>No message sent.</em>
-            </p>
+            <div class="empty">
+                <p>No e-mail messages were sent.</p>
+            </div>
         {% else %}
-            <ul class="alt">
-                {% for message in collector.messages(name) %}
-                    <li class="{{ cycle(['odd', 'even'], loop.index) }}">
-                        {% for header in message.headers.all %}
-                            <pre>{{ header }}</pre>
-                        {% endfor %}
+            {% for message in collector.messages(name) %}
+                {% if collector.messages(name)|length > 1 %}
```

Comment:
use ``loop.length`` here. It will be more readable about which length is used IMO